common_features.mk

```
@@ -356,7 +359,7 @@ endif
 ifeq ($(strip $(PRINTING_ENABLE)), yes)
     OPT_DEFS += -DPRINTING_ENABLE
     SRC += $(QUANTUM_DIR)/process_keycode/process_printer.c
-    QUANTUM_LIB_SRC += uart.c
+    SRC += $(TMK_DIR)/protocol/serial_uart.c
```

Please revert this change. 
```suggestion
    QUANTUM_LIB_SRC += uart.c
```